### PR TITLE
Load demo image if available

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: ApplicationContent.qml [hunk 1]
```diff
@@ -32,6 +32,13 @@ Item {
 		Component.onCompleted: Global.mainView = mainView
 	}
 
+	Image {
+		y: Theme.geometry_statusBar_height
+		width: parent.width
```
Comment (chriadam) [inline ApplicationContent.qml:37]: maybe set this to zero if the demo image file name is empty?

**Review by DanielMcInnes — APPROVED** [review]
Maybe add a debug message if we are displaying the demo image. I can just imagine leaving a 'real' looking demo image lying around, and spending ages trying to figure out why none of the nav buttons work

File: ApplicationContent.qml (hunk 1 above)
Comment (blammit) [reply ApplicationContent.qml]: Done

**Comment by blammit** [comment]
> Maybe add a debug message if we are displaying the demo image. I can just imagine leaving a 'real' looking demo image lying around, and spending ages trying to figure out why none of the nav buttons work

Done

**Comment by DanielMcInnes** [comment]
currently the demo image displays on every page, not just the brief page. I assume this has to change?

**Pull request merged**